third_party/java_tools/ijar/BUILD

```diff
@@ -0,0 +1,132 @@
+package(
+    default_visibility = [
+        "//visibility:public",
+    ],
+)
+
+licenses(["notice"])  # Apache 2.0
+
+cc_library(
+    name = "zip",
+    srcs = [
+        "mapped_file_unix.cc",
+        "zip.cc",
+    ],
+    hdrs = [
+        "common.h",
+        "mapped_file.h",
+        "zip.h",
+    ],
+    deps = [
+        ":platform_utils",
+        ":zlib_client",
+    ],
+)
+
+cc_library(
+    name = "zlib_client",
+    srcs = ["zlib_client.cc"],
+    hdrs = [
+        "common.h",
+        "zlib_client.h",
+    ],
+    deps = ["//third_party/java_tools/zlib"],
+)
+
+cc_library(
+    name = "platform_utils",
+    srcs = ["platform_utils.cc"],
+    hdrs = [
+        "common.h",
+        "platform_utils.h",
+    ],
+    deps = [
+        "//third_party/java_tools/src/main/cpp/util:errors",
+        "//third_party/java_tools/src/main/cpp/util:filesystem",
+        "//third_party/java_tools/src/main/cpp/util:logging",
+    ],
+)
+#
+#cc_binary(
+#    name = "zipper",
+#    srcs = ["zip_main.cc"],
+#    visibility = ["//visibility:public"],
+#    deps = [":zip"],
+#)
+
+cc_binary(
+    name = "ijar",
+    srcs = [
+        "classfile.cc",
+        "ijar.cc",
+    ],
+    deps = [":zip"],
+)
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+#filegroup(
```

Comment:
I'd say remove commented coded

Reply:
I left it commented out to have it easier to update if needed. Otherwise it's a bit harder to pick what's needed to be changed.